integrations/google_ai/pyproject.toml

```diff
@@ -56,7 +56,7 @@ cov = ["test-cov", "cov-report"]
 cov-retry = ["test-cov-retry", "cov-report"]
 docs = ["pydoc-markdown pydoc/config.yml"]
 [[tool.hatch.envs.all.matrix]]
-python = ["3.8", "3.9", "3.10", "3.11"]
+python = ["3.9", "3.10", "3.11"]
```

Comment:
unrelated: we are not declaring compatibility with python 3.8 for this package, so testing it with 3.8 makes little sense